clustered/integration-test/src/test/java/org/ehcache/clustered/writebehind/BasicClusteredWriteBehindTest.java

```diff
@@ -69,6 +80,14 @@ public static void waitForActive() throws Exception {
     CLUSTER.getClusterControl().waitForActive();
   }
 
+  @After
```

Comment:
Is this "after timeout" or just always after the test?

Reply:
:-) Always. But I will remove the code after we found out what's wrong. So I don't mind doing it all the time. We might also learn so things when the test is just failing.

Reply:
I've added a flag to dump only on lock or failure